LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2017 Daily Kos
```

Comment:
I realize _Daily Kos_ is a registered trademark of the legal entity Kos Media, LLC, but I believe we want the legal entity name here?